apps/apps.humanatlas.io/src/app/pages/landing-page/landing-page.component.html

```diff
@@ -10,8 +10,8 @@
   </div>
 </div>
 
-<div class="apps-container">
```

Comment:
Please add back the container div. This will be useful when applying the background